src/main/java/org/elasticsearch/cloud/aws/InternalAwsS3Service.java

```diff
@@ -189,6 +189,8 @@ private static String getEndpoint(String region) {
             return "s3-ap-northeast-1.amazonaws.com";
         } else if ("eu-west".equals(region) || "eu-west-1".equals(region)) {
             return "s3-eu-west-1.amazonaws.com";
+        } else if ("eu-west-2".equals(region)) {
+            return "s3-eu-west-2.amazonaws.com";
```

Comment:
Needed to add support for the London region for test purposes, might as well keep it.